### PR TITLE
Uncertified operator version

### PR DESCRIPTION
**Pull request by theemj**
Test cases (and supporting changes) for the scenario in which an operator does belong to the "certified-operators" group in the Red Hat catalog, but the version installed is not certified for the ocp version.

### CODE REVIEW

**Review by shaior — APPROVED** [review]
lgtm.

File: tests/affiliatedcertification/affiliatedcerthelper/operator.go [hunk 1]
```diff
@@ -106,3 +108,67 @@ func IsOperatorInstalled(namespace string, operatorDeploymentName string) error
...
+		},
+	)
+	if err != nil {
+		return fmt.Errorf("can not deploy catalog source %w", err)
+	}
+
+	return nil
+}
+
+func DisableCatalogSource(name string) error {
```
Comment (kononovn) [inline tests/affiliatedcertification/affiliatedcerthelper/operator.go:140]: This function looks the same as EnableCatalogSource.
You can create common private function setCatalogSource(enable bool, name string) and re-use this helper function from DisableCatalogSource and EnableCatalogSource

File: tests/affiliatedcertification/affiliatedcertparameters/parameters.go [hunk 2]
```diff
@@ -40,18 +40,19 @@ var (
...
+	TestCaseOperatorSkipRegEx                = "container-is-certified helmchart-is-certified"
+	TestCaseOperatorAffiliatedCertName       = "affiliated-certification affiliated-certification-operator-is-certified"
+	OperatorGroupName                        = "affiliatedcert-test-operator-group"
+	CertifiedOperatorGroup                   = "certified-operators"
+	CommunityOperatorGroup                   = "community-operators"
+	OperatorSourceNamespace                  = "openshift-marketplace"
+	OperatorLabel                            = map[string]string{"test-network-function.com/operator": "target"}
+	UncertifiedOperatorPrefixFalcon          = "falcon-operator"
+	UncertifiedOperatorDeploymentFalcon      = "falcon-operator-controller-manager"
+	CertifiedOperatorPrefixKubeturbo         = "kubeturbo-operator"
```
Comment (kononovn) [inline tests/affiliatedcertification/affiliatedcertparameters/parameters.go:52]: Same var as CertifiedOperatorDeploymentKubeturbo

File: tests/affiliatedcertification/tests/affiliated_certification_operator.go [hunk 3]
```diff
@@ -73,58 +73,111 @@ var _ = Describe("Affiliated-certification operator certification,", func() {
...
+
+				return !stillEnabled
+			}, affiliatedcertparameters.Timeout, affiliatedcertparameters.PollingInterval).Should(Equal(true),
+				"Default catalog source is still enabled")
+
+			err = affiliatedcerthelper.DeployRHCertifiedOperatorSource("4.5")
+			Expect(err).ToNot(HaveOccurred(), "Error deploying catalog source")
+
+			err = affiliatedcerthelper.DeployOperatorSubscription(
+				"k10-kasten-operator",
```
Comment (kononovn) [inline tests/affiliatedcertification/tests/affiliated_certification_operator.go:155]: You have this string in parameters

**Review by kononovn — CHANGES_REQUESTED** [review]
Please fix few minor comments above.

**Review by kononovn — APPROVED** [review]
Looks good

**Pull request merged**